Projects/Playground/Playground.Forms.UI/Playground.Forms.UI.csproj

```diff
@@ -19,6 +19,9 @@
     <EmbeddedResource Update="Pages\ConvertersPage.xaml">
       <Generator>MSBuild:UpdateDesignTimeXaml</Generator>
     </EmbeddedResource>
+    <EmbeddedResource Update="Pages\NavigationClosePage.xaml">
```

Comment:
Do the xaml pages need to be defined in the csproj? I thought the default behaviour is that they're included without need for them to be explicitly listed here.